docs/R-IOC/r08_ifelse.md

```diff
@@ -0,0 +1,50 @@
+## Conditions and If statements
+
+The use of conditions in R is very important. It allows us to use logical variables 
```

Comment:
Make it simple please.

A logical variable is a variable that can only take two values or levels, TRUE and FALSE.

A if statement indicate that the block code between { and } will only be executed if the condition between parenthesis is fulfilled (or TRUE, to say it in short)